### PR TITLE
update/add-chains

### PR DESCRIPTION
**Pull request by MustafaBrandon**
Add chains:
Filecoin
ApeChain

thanks,

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/steer: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'map')
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/steer/index.js:191:34)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:173:9
        at async Promise.all (index 0)
        at async Promise.all (index 27)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:216:3

File: projects/steer/index.js [hunk 1]
```diff
@@ -160,6 +160,24 @@ const supportedChains = [
...
+  },
+  // {
+  //   name: 'Bittorrent',
+  //   subgraphEndpoint: 'https://api.goldsky.com/api/public/project_clohj3ta78ok12nzs5m8yag0b/subgraphs/steer-protocol-btt/1.1.1/gn',
+  //   chainId: 199,
+  //   identifier: 'bittorrent'
+  // },
+  {
+    name: 'Filecoin',
+    subgraphEndpoint: 'https://sushi.laconic.com/subgraphs/name/sushiswap/v3-filecoin',
```
Comment (g1nt0ki) [inline projects/steer/index.js:177]: think this is wrong subgraph

**Comment by llamatester** [comment]
Error while running adapter at projects/steer: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: Llama RPC error! methos: undefined 
    - host: https://evmos-json-rpc.stakely.io error: rpc error: code = Internal desc = out of gas
    - host: https://evmos-evm.publicnode.com error: rpc error: code = Internal desc = out of gas
    - host: https://evmos-json-rpc.0base.dev error: rpc error: code = Internal desc = out of gas
    - host: https://evmos.lava.build error: rpc error: code = Internal desc = out of gas
    - host: https://evm-rpc.evmos.silentvalidator.com error: rpc error: code = Internal desc = out of gas
    - host: https://jsonrpc-evmos-ia.cosmosia.notional.ventures error: self-signed certificate
    - host: https://evmos-jsonrpc.alkadeta.com error: rpc error: code = Internal desc = out of gas

**Comment by llamatester** [comment]
The adapter at projects/steer exports TVL: 
        
 
 
    arbitrum-staking          2.16 M
    staking                   2.16 M
    polygon                   2.12 M
    linea                     2.02 M
    metis                     901.17 k
    arbitrum                  691.67 k
    kava                      685.22 k
    flare                     647.19 k
    mode                      605.37 k
    manta                     558.23 k
    base                      449.24 k
    rsk                       310.76 k
    fantom                    143.37 k
    telos                     97.98 k
    celo                      53.44 k
    optimism                  40.31 k
    astar                     15.92 k
    bsc                       9.52 k
    astrzk                    5.97 k
    blast                     5.30 k
    avax                      1.56 k
    scroll                    641
    thundercore               422
    zklink                    168
    polygon_zkevm             112
    mantle                    54
    xlayer                    22
    filecoin                  0
    apechain                  0
    
    total                    9.36 M

**Comment by MustafaBrandon** [comment]
Seems Evmos rpc is dead, just removing for now

**Comment by g1nt0ki** [comment]
I think I found a workaround for the evmos rpc, thanks for the PR

**Pull request merged**